bot/cogs/snakes.py

```diff
@@ -40,10 +62,33 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        snake = await self.get_snek(name)
+        if snake:
+            snake_embed = Embed(title=snake[0], description=snake[1])
+            snake_embed.set_image(url=snake[2])
+            await ctx.send(embed=snake_embed)
+        else:
+            await ctx.send("I was not able to find your snake, I am sorry.")
 
     # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+    @command()
+    async def hiss(self, ctx: Context):
```

Comment:
If someone executes this twice within a short period of time will there be an error, or will the sound play twice at the same time?

Reply:
You may want to check the clients voice status before making it play more.

Reply:
discordpy appears to handle this fine.